README.md

```diff
@@ -50,6 +50,18 @@ git clone https://github.com/kaitai-io/kaitai_struct_visualizer
 ksv <binary-file> <ksy-file>... | <rb-file>
 ```
 
+### Using Docker image
+
+Running image from Docker Hub:
+```shell
+docker run -v "$(pwd):/share":Z -it kaitai/ksv <binary-file> <ksy-file>
+```
+
+Rebuilding the image with newer repository sources:
```

Comment:
"If you want to update the image." might sound better, but you probably want to update Docker Hub image you're using, not the one which is uploaded, and the reason you want to update that image at all, is because you want to try run latest sources.

Reply:
> "If you want to update the image." might sound better, but you probably want to update Docker Hub image you're using, not the one which is uploaded, and the reason you want to update that image at all, is because you want to try run latest sources.

I do not understand how this is relevant. What's wrong about _Rebuilding the image from local files_? That's exactly what this command does:

```shell
docker build . --tag docker.io/kaitai/ksv
```